src/main/java/project/bot/service/KeyboardSetter.java

```diff
@@ -0,0 +1,36 @@
+package project.bot.service;
+
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardRow;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KeyboardSetter {
+
+    /**
+     * Method which initializes default keyboard for the bot
+     */
+    public ReplyKeyboardMarkup setDefaultKeyboard() {
```

Comment:
variables must be declared first. Than should be actions

Reply:
but there are no variables in this class, all of them are declared in the method

Reply:
The comment is left for the method, not class